lib/versioning.h

```diff
@@ -127,11 +128,12 @@
 #define VERSION_VALUE_3_13 0x030d
 #define VERSION_VALUE_3_14 0x030e
 #define VERSION_VALUE_3_15 0x030f
+#define VERSION_VALUE_3_16 0x0310
```

Comment:
Is this correct?

Reply:
It is correct (see `lib/tests/test_lexer.c`).

Reply:
yes, it is correct :) (I had a patch that made it possible to use `0x0316`... but it was rejected)

Reply:
I've updated an assert in test_lexer, just to make sure. :smile: